internal/learn/the-community/qanda.md

```diff
@@ -0,0 +1,98 @@
+Next level meaning:
+- no uncalculatable tx fees (fee=0)
+- super green, as there are no miners etc.
+- very robust & scalable infrastructure
+- all with EVM (more down the road).
+
+The 0 fee also enables 1000s of SC chains to easy exchange L1 assets (tokens, NFT's...) for free across chains w/o hackable bridges. 
+Fundamentally(!) change how we see DLT and how it can scale, to finally mitigate the limits of blockchain and serve real use-cases/requirements of industry/crypto/web3 better than anything built on blockchain.
+
+_Source: https://threadreaderapp.com/thread/1527201480574480385.html_
+
+# Q: Why use IOTA? No one uses it. Why not an active chain like `insert blockchains here`?
+
+# Q: Since there are no gas fees, what’s the point of iota?
+## A:
+Past coordicide you will need Mana to participate in the network. Mana is generated by holding IOTA. Nodes that do not hold IOTA, but still want to participate in the IOTA network or simply have a very high throughput demand can buy access mana via some market or you could also possibly pledge your mana to a node in return for rewards.
+https://blog.iota.org/explaining-mana-in-iota-part-2
+And while transactions are "free", smart contracts can demand a fee for their execution.
+
+_Source: https://www.reddit.com/r/Iota/comments/us4sc1/comment/i91vb5w_
+
+## A:
+
+Other applications that require the IOTA token:
+
+**Digital assets**
+IOTA is a multi-asset ledger. Most DLTs fall into the single-asset ledger category as they are only able to track ownership of a specific base currency within their ledger. Multi-asset ledgers, on the other hand, are able to manage multiple native tokens in the same ledger as the base currency. Since IOTA’s base ledger allows for feeless transactions, IOTAs multi-asset ledger is able to perform feeless transfers of any native tokens, which is a unique innovation in the field.
+
+All native tokens are standalone tokens in the ledger that are brought into existence by foundries and can be held or transferred from any account in the ledger. Native tokens are injected into the IOTA ledger by users. Therefore, they are also referred to as user-defined tokens. They consume valuable resources of the nodes that maintain the network, primarily storage space. As a result, each account holding native tokens must post a deposit in IOTA coins to compensate for the excessive resource consumption.
+
+Or in other words, storage space in the IOTA ledger costs IOTA tokens (deposit). When the storage space is no longer needed, the deposited tokens can be transferred or sold.
+
+**Micro payments**
+The absence of miners makes IOTA transactions completely charge-free transactions. Send 1 MIOTA = receive 1 MIOTA . This enables real micro payments for the machine-to-machine economy so that payment for small quantities or consumption-based payments can be implemented.
```

Comment:
```suggestion
The absence of miners makes IOTA transactions completely charge-free transactions. Send 1 MIOTA = receive 1 MIOTA . This enables real micropayments for the machine-to-machine economy so that payments for small quantities or consumption-based payments can be implemented.
```

Reply:
The micropayments thing is currently not possible anymore. Dont know if it makes sense to include it